assets/gpu-feature-discovery/0500_daemonset.yaml

```diff
@@ -36,6 +36,47 @@ spec:
             - name: run-nvidia
               mountPath: /run/nvidia
               mountPropagation: Bidirectional
+        - name: gpu-feature-discovery-imex-init
+          image: "FILLED BY THE OPERATOR"
+          command: ["/bin/bash", "-c"]
+          args:
+            - |
+              until [[ -f /run/nvidia/validations/driver-ready ]]
+              do
+                echo "waiting for the driver validations to be ready..."
+                sleep 5
+              done
+              set -o allexport
+              cat /run/nvidia/validations/driver-ready
+              . /run/nvidia/validations/driver-ready
+              
+              IMEX_NODES_CONFIG_FILE=/etc/nvidia-imex/nodes_config.cfg
+              if [[ -f /config/${IMEX_NODES_CONFIG_FILE} ]]; then
+                echo "Removing cached IMEX nodes config"
+                rm -f /config/${IMEX_NODES_CONFIG_FILE}
+              fi
+              if [[ ! -f ${DRIVER_ROOT_CTR_PATH}/${IMEX_NODES_CONFIG_FILE} ]]; then
```

Comment:
Where is DRIVER_ROOT_CTR_PATH being set?

Reply:
A few lines above when we source the `driver-ready` file.